dirty_cat/tests/test_fuzzy_join.py

```diff
@@ -201,3 +202,52 @@ def test_fuzzy_join_pandas_comparison():
     result_s_fj.rename(columns={"key_x": "key"}, inplace=True)
 
     pd.testing.assert_frame_equal(result_s, result_s_fj)
+
+
+def test_correct_encoder():
+    """Test that the encoder error checking is working as intended."""
+
+    class TestVectorizer(HashingVectorizer):
+        """Implements a custom vectorizer to check if the `encoder`
+        parameter uses the passed instance as expected.
+        Raises an error when `fit` is called.
+
+        Args:
+            HashingVectorizer (_type_): _description_
+        """
+
+        def __init__(self):
+            HashingVectorizer.__init__(self)
```

Comment:
Since we don't do anything in this `__init__`, we can leave the implementation to the parent class (so remove the method).
```suggestion
```
Also, by convention we prefer
`super().__init__(*args, **kwargs)` to `<ClassName>.__init__(self, *args, **kwargs)` (notice we also remove the `self`) :)